docs/application/native/guides/ui/tizen-ws-shell.md

```diff
@@ -0,0 +1,151 @@
+
+The TZSH provides following libraries to support each system GUI services in the Tizen:
+
+- tzsh-quickpanel: For the quickpanel window which shows notifications and system setup widgets.
+
+- tzsh-softkey: For the softkey window which shows the device back and home buttons.
+
+Most applications don't need to use the TZSH's functionalities. However, some applications may want to perform manipulation of system GUI service's window in certain situations.  For example, media player application needs to close the quickpanel window during playback of video. The tzsh-quickpanel can be used in this case.
+
+
+## Prerequisites
+
+To use the functions and structures of the tzsh-quickpanel API (in [mobile](../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) and [wearable](../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) applications), include the `<tzsh_quickpanel.h>` header file in your application:
+
+```
+#include <tzsh_quickpanel.h>
+```
+
+
+## Creating TZSH Quickpanel Handler
+Once you have created the main window for your application, call the `tzsh_quickpanel_create_with_type` function with a native window ID to create `tzsh_quickpanel_h` structure (in [mobile](../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) and [wearable](../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) applications).
+
+```
+#include <Elementary.h>
+
+static void
+init(const char *name)
+{
+   Evas_Object *main_win;
+
+   /* create main window for the application */
+   main_win = elm_win_util_standard_add(name, name);
+
+   /* set up main window */
+   ...
+   evas_object_show(main_win);
+
+
+   tzsh_h tzsh;
+   tzsh_quickpanel_h tzsh_qp;
+   tzsh_window tz_win;
+
+   /* Get native window ID of main window */
+   tz_win = elm_win_window_id_get(main_win);
+
+   /* Create tzsh_h structure */
+   tzsh = tzsh_create(TZSH_TOOLKIT_TYPE_EFL);
+
+   /* Create tzsh_quickpanel_h structure */
+   tzsh_qp = tzsh_quickpanel_create_with_type(tzsh, tz_win, TZSH_QUICKPANEL_TYPE_SYSTEM_DEFAULT);
+```
+
+
+## Showing and Hiding Quickpanel Window
+
+To show and hide quickpanel window when your application's window is activated, call the following functions:
+
+```
+   ...
+
+   if (show)
+     tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
+   else
+     tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
+
+   ...
+```
+
+## Getting Visible Status of Quickpanel Window
+To know the state of current visibility of qp service's window, call the `tzsh_quickpanel_visible_get` function. The quickpanel service's window may be visible or not visible on the screen.
+
+```
+   ...
+   tzsh_quickpanel_state_visible_e state;
+
+   tzsh_quickpanel_visible_get(tzsh_qp, &state);
+   if (state == TZSH_QUICKPANEL_STATE_VISIBLE_SHOWN)
+     /* visible state */
+     ;
```

Comment:
It looks like we don't need these ";" in line 89, line 92 and line 95.
There are comments for each cases, as like `/* visible state */`, `/* invisible state */` or `/* error */`.

Reply:
@nawab-samsung These are the comments, so the compiler ignores them as they are only for human's understanding.
The comments are used to give hints of the status, or directions such as what to do to human. As the comment is considering human will write proper code, I think only the comments without ";" would be OK.

For example in other pages on Tizen Developer Site:
- [DALi watch](https://developer.tizen.org/development/guides/native-application/application-management/applications/ui-application/dali-applications/watch-application)
![image](https://user-images.githubusercontent.com/9958374/52992620-a4e6d800-3454-11e9-8586-4d465dd12536.png)

- [EFL UI](https://developer.tizen.org/development/guides/native-application/applications/ui-application/efl-applications/basic-ui-application)
![image](https://user-images.githubusercontent.com/9958374/52992804-37877700-3455-11e9-810b-bbd8c840cc95.png)

So, isn't it better to change the comments? As an example from `/* visible state */` to `/* Do what you want when quickpanel is visible*/`

Reply:
Hello, @annie-samsung.
Would you please take a look at these conversation and give your opinion?

Reply:
I will change example code as EFL UI style without changing comments.
Thanks.